scripts/roundtrip.py

```diff
@@ -117,7 +117,7 @@ class TestRoundtrip(unittest.TestCase):
 
     def test_generator(path):
         def test(self):
-            roundtrip(self, args.rellic, path, args.clang, args.timeout, args.translate_only)
+            roundtrip(self, args.rellic, path, args.clang, args.timeout, args.translate_only, ["-O3"])
```

Comment:
Maybe you could try running `-O1` and `-O2` as well? See what it does.

Reply:
Added tests for all optimization levels